MaxText/convert_gpt3_ckpt_from_paxml.py

```diff
@@ -0,0 +1,230 @@
+"""
```

Comment:
Looks amazing but can you put this in MaxText/conversions and move llama2, mistral and gamma?

Reply:
Np.

Reply:
Done.

Reply:
Hi Rafi, I got library relative import missing  error after after moving those scripts into a sub directory `conversions`. The required library like `max_utils` can't be found by those scripts since they are not in the same folder any more. The

There are 2 possible solutions.

1. replace `python3 MaxText/llama_or_mistral_ckpt.py` with something like `cd MaxText; python -m conversions.llama_or_mistral_ckpt`. Verified it worked. I think it seems to be a hack and error-prone since we can not run those scripts elsewhere.
2. Alternative, I recommend we can mark it as `todo` once have some `setup.py`, `pyproject.toml` done in our repo. We can install it like a library, then we can import without any limitations.

What do you think and do you have better ideas?

Reply:
Yes tahnks for the explanation, that is super reasonable.